lib/SparqlJsonBindingsTransformer.ts

```diff
@@ -1,10 +1,12 @@
-import {Transform, TransformCallback} from "stream";
 import {SparqlJsonParser} from "./SparqlJsonParser";
 
+// tslint:disable-next-line:no-var-requires
+const Stream = require("readable-stream");
```

Comment:
Not nice but there is no proper typescript definitions for `readable-stream` 4.x (`@type/readable-stream` depends on NodeJS buffer API).

Reply:
Done